zone_failure/failure.sh

```diff
@@ -31,7 +31,7 @@ INSTANCE_NAME=$(hostname)
 STATE="healthy"
 while true; do
   if [[ "$ZONE" = "$(GetMetadata $PROJECT_METADATA_URL/failed_zone)" ]] && \
-     [[ "$INSTANCE_NAME" =~ "$(GetMetadata $PROJECT_METADATA_URL/failed_instance_names)" ]]; then
+     [[ "$INSTANCE_NAME" =~ $(GetMetadata $PROJECT_METADATA_URL/failed_instance_names) ]]; then
     if [[ "$STATE" = "healthy" ]]; then
```

Comment:
Maybe I should switch to glob match as recommended here: https://github.com/koalaman/shellcheck/wiki/SC2076